app/_kong_plugins/solace-consume/index.md

```diff
@@ -0,0 +1,149 @@
+---
+title: 'Solace Consume'
+name: 'Solace Consume'
+
+content_type: plugin
+tier: enterprise
+publisher: kong-inc
+description: 'Consume messages from Solace topics and make them available through HTTP endpoints'
+
+
+products:
+    - gateway
+
+works_on:
+  - on-prem
+  - konnect
+
+min_version:
+  gateway: '3.12'
+
+topologies:
+  on_prem:
+    - hybrid
+    - db-less
+    - traditional
+  konnect_deployments:
+    - hybrid
+    - cloud-gateways
+    - serverless
+
+icon: solace-consume.png
+
+categories:
+   - traffic-control
+
+tags:
+  - traffic-control
+  - events
+  - solace
+
+search_aliases:
+  - solace-consume
+  - traffic-control
+  - events
+  - event gateway
+
+related_resources:
+  - text: Solace Log plugin
+    url: /plugins/solace-log/
+  - text: Solace Upstream plugin
+    url: /plugins/solace-upstream/
+  - text: Kafka Log plugin
+    url: /plugins/kafka-log/
+  - text: Kafka Upstream plugin
+    url: /plugins/kafka-upstream/
+  - text: Confluent plugin
+    url: /plugins/confluent/
+  - text: Confluent Consume plugin
+    url: /plugins/confluent-consume/
+
+faqs:
+  - q: I'm running into SSL connection issues with my Solace broker, what can I do?
+    a: |
+      If you encounter SSL connection issues, try the following:
+
+      * Check if the Solace broker is properly configured for SSL connections
+      * Verify that the broker's SSL certificate is valid
+      * Verify that the plugin has [`config.session.ssl_validate_certificate`](./reference/#schema--config-session-ssl-validate-certificate) set to `true`
+      * Ensure the correct port is being used for SSL connections (usually port 55443)
+      
+      Note that the plugin doesn't implement any retry logic, so if the connection fails, the client has to retry sending the message.
+  - q: Connections to Solace are failing.
+    a: |
+      If connections to Solace fail, check the following:
+
+      * Network connectivity between {{site.base_gateway}} and the Solace broker
+      * Authentication credentials
+      * VPN name and permissions
+      * Review Solace broker logs for more detailed error information
+      * Increase the connection timeout on the Solace side (`solace_session_connect_timeout_ms`), or on the Kong side (see the [timeout setting](./reference/) for your mode)
+
+  - q: I'm experiencing performance issues when using the Solace Consume plugin.
+    a: |
+      If you experience performance issues with high concurrency, adjust the following settings in your Solace broker:
+
+      * Adjust the session pool size (`solace_session_pool`)
+      * Increase the `ack` timeout (`ack_max_wait_time_ms`)
+      * Consider using direct messaging instead of guaranteed messaging for higher throughput
+      
+      Since {{site.base_gateway}} workers are single-threaded, this plugin uses once Solace context per {{site.base_gateway}} worker with up to 4 sessions per plugin. 
```

Comment:
```suggestion
      Since {{site.base_gateway}} workers are single-threaded, this plugin uses one Solace context per {{site.base_gateway}} worker with up to 4 sessions per plugin. 
```
I think?